robot_sort/robot_sort.py

```diff
@@ -94,10 +94,32 @@ def light_is_on(self):
 
     def sort(self):
         """
-        Sort the robot's list.
-        """
-        # Fill this out
-        pass
+        I think its like the iterative bubble sort with the light being used as a way to iterate.  
+        I believe that going all the way back is a problem in this but couldn't figure it out
+        """
+        self.set_light_on() # start first loop
```

Comment:
Well implemented, all tests are passing. If you have time later and can improve your runtime complexity even better, but this is good enough.